ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/monitor-layer1-enabled.yml

```diff
@@ -0,0 +1,6 @@
+---
+monitor_layer1:
+  enabled: true
+  logging_mac_fault: true
+  logging_transceiver:
+    enabled: true
```

Comment:
No need of creating a new file we can use the same file input file to enable logging_transceiver

Reply:
Fixed.